cmd.sh

```diff
@@ -24,6 +33,24 @@ if [ -z "${ANACONDA_TOKEN}" ]; then
   exit -1
 fi
 
+if [ -z "${ANACONDA_LABELS}" ]; then
+  echo "ANACONDA_LABELS is empty , using default label (main)..."
+  ANACONDA_LABELS="--label main"
+else
+  # Count the number of words in the string
+  label_count=$(echo "$ANACONDA_LABELS" | wc -w)
+  if [ "$label_count" -gt 1 ]; then
+    formatted_labels=""
+    for label in ${ANACONDA_LABELS}; do
+      formatted_labels+="--label $label "
+    done
+    ANACONDA_LABELS="${formatted_labels}"
+  else
+    ANACONDA_LABELS="--label $ANACONDA_LABELS"
+    echo "only 1 label, using option: $ANACONDA_LABELS"
+  fi
+fi
```

Comment:
```suggestion
# if the ANACONDA_LABELS is empty, exit with status -1
# as this should be set in action.yml or by the user
# and it is better to fail on this to sigal a problem.
if [ -z "${ANACONDA_LABELS}" ]; then
  echo "ANACONDA_LABELS is empty, exiting..."
  exit -1
fi

# convert newlines to commas for parsing
# and ensure that there is no trailing comma
ANACONDA_LABELS="$(echo "$ANACONDA_LABELS" | tr '\n' ',' | sed 's/,$//')"

IFS=',' read -ra LABELS <<< "${ANACONDA_LABELS}"

LABEL_ARGS=""
for label in "${LABELS[@]}"; do
  LABEL_ARGS+="--label ${label} "
done
```

Okay, here I am going to suggest a total rewrite so let me explain why. At the moment the approach in the PR is assuming that the list of possible labels is space seperated. This is not standard across GitHub Actions, which in general assume that you're providing a newline seperatd or comma seperated list (c.f. https://github.com/docker/build-push-action?tab=readme-ov-file#inputs as an example).

So if we agree that we should be expecting newline seperated or comma seperated inputs then we can just treat all input the same by:
* First, converting all input into a comma separated string
* Then, create an array of the comma seperated labels
* Finally, parse that array into a single string that represents all the label arguments to be included (you were already doing this part)

This approach doesn't care about the number of labels used, and so the logic is very simple to impliment (and read in the future).

Examples of possible `ANACONDA_LABELS` inputs and the resulting `LABEL_ARGS` with the suggested changes:

* `ANACONDA_LABELS="main"`
```
LABEL_ARGS: --label main
```

* `ANACONDA_LABELS="main,dev"`
```
LABEL_ARGS: --label main --label dev
```

*
```
ANACONDA_LABELS="main
dev"
```

```
LABEL_ARGS: --label main --label dev
```

Reply:
If you also did want to allow for space deliminated lists of labels then you could just throw in a 

```bash
ANACONDA_LABELS="$(echo "$ANACONDA_LABELS" | tr ' ' ',' | sed 's/,$//')"
```

to the suggestions I made as well.